Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Godep changes look a bit wonky. I'd suggest not updating any deps that this PR does not depend on.